pkgs/by-name/he/heptabase/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  pname = "heptabase";
+  version = "1.34.2";
+  src = fetchurl {
+    url = "https://github.com/heptameta/project-meta/releases/download/v${version}/Heptabase-${version}.AppImage";
+    hash = "sha256-0wPtAvBuc8J+XJqZXDzV+RL/37NPZjKcs3do1jQkE14=";
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    install -Dm444 ${appimageContents}/project-meta.desktop -t $out/share/applications
+    cp -r ${appimageContents}/usr/share/icons $out/share
```

Comment:
The icon ends up at `results/heptabase/share/icons/hicolor/0x0/apps/project-meta.png` which has two errors:

* `project-meta` should be `heptabase`
* `0x0` should be `512x512`

Please address both and update the `Icon=` field in the desktop entry to reflect the change